src/site/artifacts/cargo_dist.rs

```diff
@@ -183,10 +184,19 @@ pub fn build(config: &Config) -> Result<Box<div<String>>> {
                     None => String::from("We couldn't detect the system you are using."),
                 };
                 let install_code_block = build_install_block(config, &typed, release, artifact);
+                let title = format!("Install v{}", release.app_version);
+                let formatted_date = match DateTime::parse_from_rfc3339(published_at.as_str()) {
+                    Ok(date) => date.format("%b %e %Y at %R UTC").to_string(),
+                    Err(_) => published_at.to_owned(),
+                };
+                let date_published = format!("Published at {}", formatted_date);
 
                 html.extend(html!(
                     <div class="hidden target artifact-header" data-targets=&targets>
-                        <h4>{text!("Install")}</h4>
+                        <h4>{text!(title)}</h4>
```

Comment:
per my comment above i think we should add all this logic to the "install block" function

Reply:
done